pkgs/applications/editors/amp/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, rustPlatform, openssl, pkgconfig, python3, xorg, cmake, libgit2 }:
+
+rustPlatform.buildRustPackage {
+  name = "amp";
+  # The latest release (0.5.2) does not compile, so we use a git snapshot instead.
+  version = "2019.06.11";
```

Comment:
Please change to `unstable-2019-06-11`.

Reply:
There are many other packages that don't follow that convention, e.g. yosys.

Reply:
Also, with your convention, if you combine ``${pname}-${version}`` then Nix won't recognize ``unstable`` as part of the version number as it does not start with a number.

Reply:
https://nixos.org/nixpkgs/manual/
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

Reply:
Ok...

Reply:
@sbourdeauducq The date should refer to the date of the commit you're referencing, not the date added to nixpkgs, so `2019-06-09` in this case.

Reply:
Changed